src/components/Asset/AssetContent/index.tsx

```diff
@@ -78,6 +79,11 @@ export default function AssetContent({
               </Button>
             </div>
           )}
+          <RelatedAssets
+            tags={asset?.metadata?.tags}
```

Comment:
As this component conceptionally will always be used within the `Asset` context, there is no need to pass those props. Remove them and use the `useAsset()` hook to get them from inside the `RelatedAssets` component